bemani/utils/read.py

```diff
@@ -1499,22 +1563,39 @@ def import_music_db(self, songs: List[Dict[str, Any]]) -> None:
 
             self.start_batch()
             for chart in self.charts:
-                # First, try to find in the DB from another version
-                old_id = self.get_music_id_for_song(songid, chart)
-                if self.no_combine or old_id is None:
-                    # Insert original
-                    print(f"New entry for {songid} chart {chart}")
-                    next_id = self.get_next_music_id()
+                if(chart <= 2):
```

Comment:
I'm not quite sure I follow the logic here. Shouldn't there be something checking for whether the game is festo or above and not be inserting charts 3-5 for clan and below?